### ISSUE TITLE
Small event power records

### ISSUE THREAD

**Issue by nickw29** [nottinghamtec/PyRIGS#612]
In line with the policy, the "Equipment appropriately earthed?" check box should be replaced with one labelled "Public/Performer accessible circuits tested? (using socket tester)"

**Comment by jb3** [comment]
This is a simple change, the important thing here is that it isn't just a rename otherwise we retrospectively update all power test records which obviously is probably not something we want to do.

I can have a look at this next week at some point unless someone else wants to, but just bear that in mind that we shouldn't make a change which changes too much about existing power test records.

**Comment by FreneticScribbler** [comment]
Add a new field, hide the old one from the form (and from display, if its blank) sorted no? 

Though isn't the latter checkbox something that should exist on all power record forms, not just small?

**Comment by jb3** [comment]
> Add a new field, hide the old one from the form (and from display, if its blank) sorted no? 
> 
> Though isn't the latter checkbox something that should exist on all power record forms, not just small? 

It is easy yep, I'm just saying in case a new member wanted to take it on that it's not a "straight" replacement.

Yes, it is already a field we capture on larger events.

**Comment by nickw29** [comment]
Yes we don't want to alter previous records so I agree a new field and hide the old one is the way to go.

It does already exist on the full checklist, but whilst we are here I'll double check everything else is as it should be on both

**Comment by FreneticScribbler** [comment]
Is it not possible for an event needing "earthing" (that should definitely say bonding, come to think of it) to also be small? Staging, for example?

Or does the requirement for any supplementary bonding kick it up to Medium automatically? My "TEC Policy" mental database has been purged to save disk space...

**Comment by jb3** [comment]
Policy is additional earthing automatically makes an event a Medium power rig.

Interestingly, whilst that's in the policy file it doesn't seem to be replicated on RIGS in the risk assessments or event size determinations.

Staging only needs additional earthing past 0.5m so for most rigs the threshold isn't crossed.

**Comment by nickw29** [comment]
Yes, we don't have a question on the RA regarding additional earthing or bonding, but tbh I can't think of a time we've had a stage over 0.5m on a 'small' rig but something we can consider.

> Is it not possible for an event needing "earthing" (that should definitely say bonding, come to think of it) to also be small? Staging, for example?

I also noticed this, you are right. It should probably say "additional earthing or bonding".
